pkg/apis/serving/v1beta1/predictor_tfserving.go

```diff
@@ -83,11 +85,18 @@ func (t *TFServingSpec) GetStorageUri() *string {
 
 // GetContainers transforms the resource into a container spec
 func (t *TFServingSpec) GetContainer(metadata metav1.ObjectMeta, extensions *ComponentExtensionSpec, config *InferenceServicesConfig) *v1.Container {
+	// Get the timeout from user input or else use the default timeout
+	if extensions.TimeoutSeconds != nil {
+		TimeoutMilliSeconds = 1000 * *extensions.TimeoutSeconds
+	} else {
+		TimeoutMilliSeconds = 1000 * constants.DefaultPredictorTimeout
```

Comment:
should get the configured default value instead of the constant?

Reply:
indeed that should have been the case, I've fixed it in 2c3b5de